src/NuGet.Packaging/PackagesConfigReader.cs

```diff
@@ -122,11 +123,21 @@ public NuGetVersion GetMinClientVersion()
         }
 
         /// <summary>
-        /// Reads all package node entries in the config
+        /// Reads all package node entries in the config.
+        /// If duplicate package ids exist an exception will be thrown.
         /// </summary>
-        /// <returns></returns>
         public IEnumerable<PackageReference> GetPackages()
         {
+            return GetPackages(allowDuplicatePackageIds: false);
```

Comment:
No - that's the default behavior. There is a similar pr in nuget.package.management that covers the restore command

Sent from my Windows Phone
________________________________
From: daazose<mailto:notifications@github.com>
Sent: ‎9/‎29/‎2015 1:04 PM
To: NuGet/NuGet3<mailto:NuGet3@noreply.github.com>
Cc: Yishai Galatzer<mailto:yigalatz@microsoft.com>
Subject: Re: [NuGet3] PackagesConfigReader support for duplicate package id entries (part 1) (#134)


In src/NuGet.Packaging/PackagesConfigReader.cs<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fNuGet%2fNuGet3%2fpull%2f134%23discussion_r40721628&data=01%7c01%7cyigalatz%40microsoft.com%7cb17b55ee91ca4dc8d42708d2c9093b48%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=7bHgtC6Cidi89G2ZEG%2fHyHNLP5yHXGHd%2f41TKkyL6bE%3d>:

>          public IEnumerable<PackageReference> GetPackages()
>          {
> +            return GetPackages(allowDuplicatePackageIds: false);


@yishaigalatzer<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fyishaigalatzer&data=01%7c01%7cyigalatz%40microsoft.com%7cb17b55ee91ca4dc8d42708d2c9093b48%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=vj5BQvSFoAA1bMQrJHd8ZqEeDMkSQIluXCKqIK4qa5M%3d>, @emgarten<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2femgarten&data=01%7c01%7cyigalatz%40microsoft.com%7cb17b55ee91ca4dc8d42708d2c9093b48%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=rMliXrHYQj9R%2f0OHGJI%2fql5v4eqeZQ8QJGZ6oskjhuc%3d> - should this be set to true given the response below?

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fNuGet%2fNuGet3%2fpull%2f134%2ffiles%23r40721628&data=01%7c01%7cyigalatz%40microsoft.com%7cb17b55ee91ca4dc8d42708d2c9093b48%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=5jScWZu3ovkrjITUUH1Npzn2qrw0W72tFGs%2br%2fiTwsU%3d>.